src/Api/AdminConsole/Services/GroupsControllerService.cs

```diff
@@ -0,0 +1,319 @@
+        IUpdateGroupCommand updateGroupCommand,
+        IDeleteGroupCommand deleteGroupCommand,
+        IAuthorizationService authorizationService,
+        IApplicationCacheService applicationCacheService,
+        IUserService userService,
+        IFeatureService featureService,
+        IOrganizationUserRepository organizationUserRepository,
+        ICollectionRepository collectionRepository)
+    {
+        _groupRepository = groupRepository;
+        _groupService = groupService;
+        _organizationRepository = organizationRepository;
+        _currentContext = currentContext;
+        _createGroupCommand = createGroupCommand;
+        _updateGroupCommand = updateGroupCommand;
+        _deleteGroupCommand = deleteGroupCommand;
+        _authorizationService = authorizationService;
+        _applicationCacheService = applicationCacheService;
+        _userService = userService;
+        _featureService = featureService;
+        _organizationUserRepository = organizationUserRepository;
+        _collectionRepository = collectionRepository;
+    }
+
+    /// <summary>
+    /// Gets the basic group information of an organization
+    /// </summary>
+    /// <param name="orgId">Organization id/param>
+    /// <param name="groupId">Group id</param>
+    /// <returns>GroupResponseModel</returns>
+    public async Task<GroupResponseModel> GetOrganizationGroup(string orgId, string groupId)
+    {
+        var group = await _groupRepository.GetByIdAsync(new Guid(groupId));
+        if (group == null || !await _currentContext.ManageGroups(group.OrganizationId))
+        {
+            throw new NotFoundException();
+        }
+
+        return new GroupResponseModel(group);
+    }
+
+    /// <summary>
+    /// Gets the detailed group information of an organization
+    /// </summary>
+    /// <param name="orgId">Organization id</param>
+    /// <param name="groupId">Group id</param>
+    /// <returns>GroupDetailsResponseModel</returns>
+    public async Task<GroupDetailsResponseModel> GetOrganizationGroupDetail(string orgId, string groupId)
+    {
+        var groupDetails = await _groupRepository.GetByIdWithCollectionsAsync(new Guid(groupId));
+        if (groupDetails?.Item1 == null || !await _currentContext.ManageGroups(groupDetails.Item1.OrganizationId))
+        {
+            throw new NotFoundException();
+        }
+
+        return new GroupDetailsResponseModel(groupDetails.Item1, groupDetails.Item2);
+    }
+
+    /// <summary>
+    /// Gets the detailed information on all groups in an organization
+    /// </summary>
+    /// <param name="user">
+    ///     Requesting user. This user must have the ability to 
+    ///     view all groups in the organization</param>
+    /// <param name="orgId">Organization id</param>
+    /// <returns>List of GroupDetailsResponseModel</returns>
+    public async Task<IEnumerable<GroupDetailsResponseModel>> GetOrganizationGroupsDetails(ClaimsPrincipal user, Guid orgId)
```

Comment:
Returning `IEnumerable<GroupDetailsResponseModel>' instead of `ListResponseModel<GroupDetailsResponseModel>` the Controller will return the `ListResponseModel` 

This can be changed back if it's determined to be better.